### DISCUSSION TITLE
3.12.14: CLI status shows out of support

### DISCUSSION THREAD

**Discussion by nesonger** [rabbitmq/rabbitmq-server#11493] [Questions] [answered]
After upgrading from 3.12.8 or 3.12.13 to 3.12.14, the CLI status is showing "RabbitMQ release series support status: out of support". Before the upgrade it was showing supported. Is this going to cause any issues? This is with RabbitMQ installed on a Windows Server 2022.

**Comment by lukebakken** [comment, marked as answer]
These links will be helpful:
* https://www.rabbitmq.com/release-information
* https://github.com/rabbitmq/rabbitmq-server/blob/main/COMMUNITY_SUPPORT.md
